src/components/pullRequests/card/card.module.scss

```diff
@@ -0,0 +1,77 @@
+.card {
+  border: 2px solid black;
+  border-radius: 10px;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  margin-bottom: 30px;
+  margin-left: 60px;
+  margin-right: 60px;
+}
+
+.prTitle {
+  color: #041484;
+  font-size: 25px;
+  font-weight: bold;
+}
+
+.cardFooter {
+  display: flex;
+  flex-direction: row;
```

Comment:
NIT : By default when you use `display: flex` the direction is `row` . so `flex-direction: row` is not necessary.